deepchem/models/keras_model.py

```diff
@@ -131,7 +120,7 @@ def __init__(self,
                learning_rate: Union[float, LearningRateSchedule] = 0.001,
                optimizer: Optional[Optimizer] = None,
                tensorboard: bool = False,
-               wandb: bool = False,
+               wandb_logger: Optional[WandbLogger] = None,
```

Comment:
For backwards compatibility, we would need to continue supporting the previous flag as well. You can add a deprecation warning on it (we can only make breaking changes at the major release version so can't remove it till DeepChem 3)

Reply:
One thought is that if `wandb` is set, we instantiate a default logger here

Reply:
Sure, I've re-add support for the wandb flag back into fit() and ValidationCallback along with deprecation warning.